packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCard.base.tsx

```diff
@@ -1,41 +1,59 @@
 import * as React from 'react';
-import { IChartHoverCardStyles, IChartHoverCardStyleProps, IChartHoverCardProps } from './ChartHoverCard.types';
+import {
+  IChartHoverCardStyles,
```

Comment:
I don't see the hover card design updated in Line, Area, Vertical Bar charts

Reply:
pushed the changes